### ISSUE TITLE
ML: validation loss slope

### ISSUE THREAD

**Issue by moellep** [radiasoft/sirepo#2984]
When training, the slope of the validation loss plot (over X number of epochs) should be shown, possibly as a table at the bottom of the Training Error plot. A positive slope indicates overtraining.
